crates/sui-framework/sources/Bag.move

```diff
@@ -118,16 +118,28 @@ module Sui::Bag {
     }
 
     /// Remove the object from the Bag, and then transfer it to the signer.
-    public(script) fun remove_and_take<T: key>(c: &mut Bag, object: T, ctx: &mut TxContext) {
+    public(script) fun remove_and_take<T: key + store>(c: &mut Bag, object: T, ctx: &mut TxContext) {
         let object = remove(c, object);
         Transfer::transfer(object, TxContext::sender(ctx));
     }
 
     /// Transfer the entire Bag to `recipient`.
-    public(script) fun transfer(c: Bag, recipient: address, _ctx: &mut TxContext) {
+    public(script) fun transfer_(c: Bag, recipient: address) {
         Transfer::transfer(c, recipient)
     }
 
+    /// Transfer the entire Bag to `recipient`.
+    public fun transfer(c: Bag, recipient: address) {
+        Transfer::transfer(c, recipient)
+    }
```

Comment:
It took me a while to realize the difference between these two functions (one is entry, and one is not). The difference is very subtle. Worth with either an explicitly comment or some better naming convention to highlight the difference.

Reply:
I can do that, but I also want to fix this rather quickly (probably early next week). Where entry != visibility.